packages/common/src/hooks/useIsScreen/useIsExtraLargeScreen.ts

```diff
@@ -1,7 +1,6 @@
-import { useMediaQuery, useTheme } from '@mui/material';
+import { useTheme, useMediaQuery } from '@mui/material';
 
 export const useIsExtraLargeScreen = (): boolean => {
   const theme = useTheme();
-
   return useMediaQuery(theme.breakpoints.up('xl'));
```

Comment:
this one should use `useIsScreen` as well?

Reply:
the confusion is at a maximum!

`useIsScreen` is using `down`, so `useIsLargeScreen` is `down, xl` - where as extra large is `up` - anything above xl..